src/connectors/dnalounge.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const DNA_PREFIX = 'DNA Pizza Music Videos: ';
+
+// The Artist+Track is displayed in the text of this Element
+Connector.playerSelector = '#metadata';
```

Comment:
You should use other selector as a `playerSelector`, otherwise, play/pause won't work. `.video_embed` which is used to check playing state, should be a child of a `playerSelector` element.